pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -450,9 +450,13 @@ func (c *Controller) networkFromMeshNetworks(endpointIP string) network.ID {
 	}
 
 	if c.ranger != nil {
-		entries, err := c.ranger.ContainingNetworks(net.ParseIP(endpointIP))
+		ip := net.ParseIP(endpointIP)
+		if ip == nil {
+			return ""
```

Comment:
what does this mean?

Reply:
when can net.ParseIP(endpointIP) == nil  happen? I mean why it can fix the original issue

Reply:
https://github.com/istio/istio/blob/276fb292f30e3f20cc036fabba6328b76291c781/pilot/pkg/serviceregistry/serviceentry/workloadentry.go#L34 - I guess it can be empty.

Reply:
We can check in that function. Checking it here would handle any such calls in future

Reply:
IC, make sense to me